code/src/index.css

```diff
@@ -1,13 +1,121 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+  font-family: "Roboto", -apple-system, BlinkMacSystemFont, "Segoe UI", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
-
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+.message-wrapper {
+  margin: 0;
+  padding: 30px 0;
+  width: 300px;
+  max-width: 500px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+.send-message-card {
+  display: block;
+  width: 100%;
+  height: 150px;
+  background-color: #F2F0F0;
+  border: 1px solid #858585;
+  box-shadow: 5px 5px #000000;
+  padding: 15px; 
+  font-size: 20px;
+  margin: 10px;
+  resize: none;
+}
+.input-field { 
+  width: 100%;
+  overflow-wrap: break-word; 
+  background-color: #ffffff;
+  padding: 5px 5px 20px 5px;
+  border: 1px solid #dedcdc;
+  font-family:"Roboto";
+  font-size: 20px; 
+  box-sizing: border-box;
+  margin-top: 10px;
+  resize: none;
+}
+.submit-button {
+  background-color: #FFADAD;
+  border-radius: 20px;
+  border: none;
+  font-family: "Roboto";
+  font-size: 15px;
+  margin: 10px 0 30px 0;
+  padding: 5px;
+  color: #442E2E;
+  cursor: pointer;
+}
+.submit-button:hover {
+  opacity: 0.7 ;
+}
+.received-message-container {
+  margin: 20px 10px;
+  width: 100%;
+  height: auto;
+  overflow-wrap: break-word; /* here too? */
+  background-color: #ffffff;
+  border: 1px solid #858585;
+  font-family:"Roboto";
+  box-shadow: 5px 5px #000000;
+  padding: 15px;
+  font-size: 20px;
+  word-wrap: break-word;
+}
+.received-message{
+  margin: 0;
+  padding-bottom:30px;
+}
+.container-date-and-likes {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+.date {
+  font-size: 12px;
+  color: #CDCDCD;
+  float: inline-end;
+  padding-left: 130px;
+}
+.heart-like-button {
+  width: 30px;
+  height: 30px;
+  border-radius: 50%;
+  border: none; 
+  background-color: #EAEAEA;
+  cursor: pointer;
+}
+.pink-heart {
+  background-color: #FFADAD;
+}
+.like-counter {
+  font-size: 12px;
+  padding-left: 10px;
+  color: #ABABAB;
+}
+.characters-counter {
+  float: right;
+  color: #858585;
+  font-size: 10px;
+}
+.red-counter {
+  color: rgb(210, 70, 70);
+}
+.page-container {
+  background-color: #858585;
+
+}
+
+@media (max-width: 1200px) {
+  .received-message-container {
+  width: 300px;
+  }
```

Comment:
I really liked you used classes for each element you styled and that the names of your classes help to understand easily what you are styling. 
It might be just a personal preference but leaving one row between each element makes the code a bit easier to read. But at the same time this way you have kept your code shorter which can also be appreciated by fellow programmers. 😄